Documentation/Using-Umbraco/Config-files/umbracoSettings/index.md

```diff
@@ -16,7 +16,7 @@ Below you can see settings that affects content in Umbraco.
 This section is used for managing thumbnail creation, allowed attributes and, which properties of an image that should be automatically updated on upload.
 
         <imaging>
-            <!-- what file extension that should cause umbraco to create thumbnails -->
+            <!-- what file extension that should cause Umbraco to create thumbnails -->

```

Comment:
Not sure about correcting these code extracts from the config files (more below)...?